access/slack/INSTALL.md

```diff
@@ -151,15 +151,22 @@ certificates there and make sure the config points to them.
 In Slack section, use the OAuth token provided by Slack.
 
 ```TOML
-# Example Teleport Slack Plugin config file
+# example slack plugin configuration TOML file
+
 [teleport]
-auth_server = "example.com:3025"  # Teleport Auth Server GRPC API address
-client_key = "/var/lib/teleport/plugins/slack/auth.key" # Teleport GRPC client secret key
-client_crt = "/var/lib/teleport/plugins/slack/auth.crt" # Teleport GRPC client certificate
-root_cas = "/var/lib/teleport/plugins/slack/auth.cas"   # Teleport cluster CA certs
+auth_server = "0.0.0.0:3025"                              # Teleport Auth Server GRPC API address
+
+# tctl auth sign --format=file --auth-server=auth.example.com:3025 --user=access-plugin --out=auth --ttl=1h
+identity = "/var/lib/teleport/plugins/slack/auth"         # Teleport certificate ("file" format)
+
+# tctl auth sign --format=tls --auth-server=auth.example.com:3025 --user=access-plugin --out=auth --ttl=1h
+# client_key = "/var/lib/teleport/plugins/slack/auth.key" # Teleport GRPC client secret key ("tls" format")
+# client_crt = "/var/lib/teleport/plugins/slack/auth.crt" # Teleport GRPC client certificate ("tls" format")
+# root_cas = "/var/lib/teleport/plugins/slack/auth.cas"   # Teleport cluster CA certs ("tls" format")
 
 [slack]
-token = "api-token"       # Slack Bot OAuth token
+token = "xoxb-11xx"                                # Slack Bot OAuth token
+recipients = ["person@email.com","YYYYYYY"]        # Optional Slack Rooms
```

Comment:
`recipients` list is the old mode where recipients are set statically in config file.

Normally, people should specify "suggested reviewers" when they request the role. If any of the suggested reviewers match the slack user emails — that users will be notified.

Reply:
This is why `recipients` is not a required option.

Reply:
Ah, I was unaware that's the recommended field to configure. Will update, thanks!